lib/mpl_toolkits/axisartist/axis_artist.py

```diff
@@ -105,13 +105,14 @@ def get_attribute_from_ref_artist(self, attr_name):
 
 class Ticks(AttributeCopier, Line2D):
     """
-    Ticks are derived from Line2D, and note that ticks themselves
+    Ticks are derived from `~.Line2D`, and note that ticks themselves
     are markers. Thus, you should use set_mec, set_mew, etc.
 
     To change the tick size (length), you need to use
-    set_ticksize. To change the direction of the ticks (ticks are
+    `set_ticksize`. To change the direction of the ticks (ticks are
     in opposite direction of ticklabels by default), use
-    set_tick_out(False).
+
+      set_tick_out(False)
```

Comment:
```suggestion
    ``set_tick_out(False)``.
```

I'm not sure, what you want to do here, but I'm pretty sure, this is not correct. (can't check because the build docs have already expired).